eph-es/manifest.yaml

```diff
@@ -0,0 +1,25 @@
+---
+applications:
+  - name: eph-es
+    routes:
+      - route: identity-idva-es-((ENVIRONMENT)).apps.internal
+    health-check-type: process
```

Comment:
Is this applicable to our cloud.gov ES app? https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html

Reply:
sure. es does listen on http so we could use http health check on any path. i don't see a big differance.

Reply:
i moved it to a port check.